### PR TITLE
Add instructions for advanced logging with torrc

### PR DESCRIPTION
**Pull request by RotationMatrix**
will-fix: https://dip.torproject.org/web/support/issues/15

### CODE REVIEW

File: content/tbb/tbb-21/contents.lr [hunk 1]
```diff
@@ -8,3 +8,8 @@ description:
 Click the button labelled "Copy Tor Log To Clipboard" that appears in the dialog window when Tor Browser is first connecting to the network.
 If Tor Browser is already open, click on the Torbutton icon (the small green onion at the top-left of the screen), then "Open Network Settings", then "Copy Tor Log To Clipboard".
 Once you have copied the log, you will be able to paste it into a text editor or email client.
+
+#### Advanced
+You can control how Tor logs information with the `Log` setting in your torrc file. Have a look at some common uses in [this torrc sample](https://gitweb.torproject.org/tor.git/tree/src/config/torrc.sample.in#n30) and the [Tor man page](https://2019.www.torproject.org/docs/tor-manual.html.en#Log) documentation for `Log`.
+
+Not sure how to edit your torrc? See [this question](https://2019.www.torproject.org/docs/faq#torrc) to learn how!
```
Comment (gusgustavo) [inline content/tbb/tbb-21/contents.lr]: We should stop ourselves from redirecting new documentation to the old TPO world. If that resource is important and still works, we should import to new TPO world (support, community, tpo, or hold to dev.tpo or docs.tpo). 
Some FAQ questions should be imported to support.tpo

**Comment by RotationMatrix** [comment]
There was some discussion on IRC yesterday about whether we should have potentially dangerous information like this in support. At very least, if we do put it in support, we need to warn (read scare) the user about the de-anonymization and security risk.

**Comment by gusgustavo** [comment]
Ok, I'm reviewing this question.

I frequently use this answer to reply to users on our support channel. So far, I didn't need to ask them to edit torrc. I'm afraid on showing more options in this question, people will do silly mistakes trying to fix their problem...

So, following our IRC discussion, we could migrate this FAQ question as a new question:
https://2019.www.torproject.org/docs/faq#torrc
And say how is a really bad idea to edit your torrc if you're following random advice on the internet.

What do you think?

**Comment by RotationMatrix** [comment]
I'm good with that. To make sure I'm understanding you, we want to remove the "Advanced" section from "How do I view Tor Browser message log?" and migrate https://2019.www.torproject.org/docs/faq#torrc to support with a security/de-anonymization warning?

**Comment by RotationMatrix** [comment]
I've implemented the changes detailed above. Please let me know if there's anything you'd like changed!

Tasks remaining before merge:
- [x] Confirm Windows instructions
- [x] Confirm Linux instructions
- [x] Squash commits

**Comment by gusgustavo** [comment]
Hey @RotationMatrix, I really liked this answer! I'm just waiting for the last tasks to merge :smiley:

**Pull request merged**